Library/Homebrew/test/test_bottle_versions.rb

```diff
@@ -75,4 +75,12 @@ def test_ngircd_style
     assert_version_detected '22',
       'ngircd-22.mavericks.bottle.tar.gz'
   end
+
+  def test_apparix_style
+    assert_version_detected '11-062',
+      'apparix-11-062.yosemite.bottle.tar.gz'
+
+    assert_version_detected '11-062',
+      'apparix-11-062..bottle.tar.gz'
```

Comment:
Rethink a bit. I think we can try to remove `.<tag>.bottle.tar.gz` in the very beginning of `BottleVersion._parse`. Maybe something like this:

```ruby
stem.gsub! /.[a-z_]*.bottle.tar.gz$/, ""
```